test/jdk/tools/jextract/JextractToolProviderTest.java

```diff
@@ -37,6 +37,7 @@
  * @test
  * @modules jdk.incubator.jextract
  * @build JextractToolRunner
+ * @bug 8240181
  * @run testng/othervm -Duser.language=en --add-modules jdk.incubator.jextract JextractToolProviderTest
  */
 public class JextractToolProviderTest extends JextractToolRunner {
```

Comment:
One comment on testing - note that we also have a separate framework to just test the API (w/o jextract). After all, this is not, strictly speaking a jextract bug, but a bug in the declaration API parsing, so perhaps a more direct test would be preferrable?

Reply:
Agree. This test is ported from older jextract tests (doesn't mean that we've to use the same  test holder class). I'll fix this.

Reply:
On extract API testing. I'll address that under another bug.